bundles/AdminBundle/config/pimcore/config.yaml

```diff
@@ -1,3 +1,10 @@
 imports:
   - { resource: default.yaml }
-  - { resource: two_factor_authentication.yaml }
\ No newline at end of file
+  - { resource: two_factor_authentication.yaml }
+
+monolog:
+    handlers:
+        error:
+            type: stream
```

Comment:
This seems to be unrelated to this PR, right?

Reply:
You are right, sorry for that. It is reverted now.